npm/vite-dev-server-fresh/cypress/e2e/react.cy.ts

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+/// <reference path="../support/e2e.ts" />
+import type { e2eProjectDirs } from '@packages/frontend-shared/cypress/e2e/support/e2eProjectDirs'
+
+type ProjectDirs = typeof e2eProjectDirs
+
+const VITE_REACT: ProjectDirs[number][] = ['vite2.8.6-react', 'vite2.9.1-react']
+
+// Add to this list to focus on a particular permutation
+const ONLY_PROJECTS: ProjectDirs[number][] = []
+
+afterEach(() => {
+  // Visiting about.blank to cleanup any sockets left over between each test,
+  // such as __vite_ping
```

Comment:
this should happen in a `beforeEach`, not `afterEach`, otherwise the page is wiped each time.

Also its unclear why this is necessary - won't this be a problem for all users if we have to do this?

Reply:
Good to know

Reply:
Added `experimentalSessionSupport` to the cy config